peachjam/js/components/document-content.ts

```diff
@@ -59,6 +59,13 @@ class DocumentContent {
       this.pdfRenderer = new PdfRenderer(root);
       this.pdfRenderer.onPreviewPanelClick = () => { this.navOffCanvas?.hide(); };
       this.pdfRenderer.onPdfLoaded = () => {
+        const searchInput: HTMLInputElement | null = this.root.querySelector('[aria-label="Search document content"]');
+        const searchForm: HTMLFormElement | null = this.root.querySelector('.doc-search__form');
+        if (searchInput) {
+          searchInput.value = 'law';
```

Comment:
this is a hard-coded value. I'm guessing you were using this for testing purposes. Please set `search.value` to the  `url param` `q`